src/support/parse5-utils.ts

```diff
@@ -0,0 +1,253 @@
+/**
```

Comment:
Isn't this already a package?

Reply:
There's a package called dom5 that hasn't been updated to use latest parse5 typings because monorepo.